deepspeed/runtime/zenflow/zenflow_config.py

```diff
@@ -39,6 +39,10 @@ class ZenFlowConfig(DeepSpeedConfigModel):
     full_warm_up_rounds: int = 0
     """Number of initial rounds during which all gradients are fully updated (no selection)."""
 
+    pt_reserved_cores: int = 1
```

Comment:
@Antlera, so you are suggesting that this should be something like?

```suggestion
    pt_reserved_cores: int = ceil(num_cores/2)
```

Reply:
If we wish to reserve half of the cores by default, a parameter called `pt_reserved_cores_perc` would be better choice.  The default value could be `0.5`.